src/main/java/com/google/api/codegen/transformer/java/JavaSurfaceNamer.java

```diff
@@ -144,4 +147,25 @@ public String getPagedResponseTypeInnerName(
   public String getFullyQualifiedApiWrapperClassName(Interface service) {
     return getPackageName() + "." + getApiWrapperClassName(service);
   }
+
+  @Override
+  public String getRandomStringValue(String randomString) {
+    String randomValueGen = "System.currentTimeMillis()";
+    String delimiter = ",";
+    String[] splitted =
+        FieldStructureParser.stripQuotes(randomString)
```

Comment:
The problem is more with the dependency on `FieldStructureParser` from `JavaSurfaceNamer`. The `stripQuotes` function should probably be moved to a more general place.